lib/screens/home_page.dart

```diff
@@ -15,7 +16,12 @@ class _HomePageState extends State<HomePage> {
     return Scaffold(
       appBar: HomePageAppBar(),
       drawer: CustomDrawer(),
-      body: MapWidget(),
+      body: Stack(
+        children: <Widget>[
+          MapWidget(),
```

Comment:
Looks like mapwidget might be getting a bit too crowded again. Might be worth separating some items and extracting some widgets out of it.